addins/Cake.NSwag.yml

```diff
@@ -2,8 +2,9 @@ Name: Cake.NSwag
 NuGet: Cake.NSwag
 Assemblies:
 - "/**/Cake.NSwag.dll"
-Repository: https://github.com/agc93/Cake.NSwag
+Repository: https://github.com/agc93/Cake.NSwag/
 Author: Alistair Chapman
 Description: "Cake addin for compiling client code and API definitions (including Swagger) using the NSwag toolchain. If you have issues running this addin on .NET Core, try using Cake.NSwag.Console instead."
+Prerelease: "true"
```

Comment:
@Jericho Wonder if this is correct to switch to pre-release just because an newer pre-release exists?

Reply:
As mentioned previously, I was in favor of ignoring pre-release addins but @gep13 thought otherwise.